src/common/datasource/src/file_format.rs

```diff
@@ -28,13 +28,15 @@ use arrow::record_batch::RecordBatch;
 use arrow_schema::{ArrowError, Schema};
 use async_trait::async_trait;
 use bytes::{Buf, Bytes};
+use datafusion::arrow::datatypes::Schema as ArrowSchema;
```

Comment:
Isn't `datafusion::arrow::datatypes::Schema` and `arrow_schema::Schema` the same type?